cmake/common.cmake

```diff
@@ -405,11 +405,10 @@ function(add_e2e_test)
       TEST ${PARSED_ARGS_NAME} APPEND
       PROPERTY ENVIRONMENT "PYTHONPATH=${CCF_DIR}/tests:$ENV{PYTHONPATH}"
     )
-    if(${PARSED_ARGS_IS_SUITE})
-      set_property(TEST ${PARSED_ARGS_NAME} APPEND PROPERTY LABELS suite)
-    else()
-      set_property(TEST ${PARSED_ARGS_NAME} APPEND PROPERTY LABELS end_to_end)
-    endif()
+    set_property(TEST ${PARSED_ARGS_NAME} APPEND PROPERTY LABELS end_to_end)
```

Comment:
This is to give us the freedom to split up stuff that doesn't parallelise well, like tests that do a lot of quote verification.